src/BulletDynamics/Featherstone/btMultiBody.h

```diff
@@ -616,7 +616,7 @@ void addJointTorque(int i, btScalar Q);
     void operator=(const btMultiBody &);  // not implemented
 
 
-	void solveImatrix(const btVector3& rhs_top, const btVector3& rhs_bot, float result[6]) const;

```

Comment:
it is a left-over from older code before we introduced spatial algebra. We can remove it.